view/frontend/web/js/view/catalog/badge.js

```diff
@@ -53,24 +57,32 @@ define([
         }
     }
 
-    function moveToCustomPosition(customDisplay,containerId){
+    /**
+     *
+     * @param customDisplay Widget config define in view/frontend/templates/catalog/product/view.phtml
+     * @param baseContainerId Base container who contain the widget
+     */
+    function moveToCustomPosition(customDisplay,baseContainerId){
```

Comment:
can you add ` ` (space) after `,` (comma) please  😉

Reply:
Done